engine/src/main/resources/application.properties

```
@@ -53,6 +53,13 @@ mp.messaging.incoming.fromcamel.group.id=integrations
 mp.messaging.incoming.fromcamel.key.deserializer=org.apache.kafka.common.serialization.StringDeserializer
 mp.messaging.incoming.fromcamel.value.deserializer=org.apache.kafka.common.serialization.StringDeserializer
 
+# Input queue for the "export requests" coming from the export service.
+mp.messaging.incoming.export-requests.connector=smallrye-kafka
+mp.messaging.incoming.export-requests.topic=platform.export.requests
+mp.messaging.incoming.export-requests.group.id=export-requests
```

Let's stick with the same group ID we use for all other topics.
```suggestion
mp.messaging.incoming.export-requests.group.id=integrations
```

---

Done!